ui/apps/platform/src/services/NetworkService.js

```diff
@@ -276,9 +276,7 @@ export function fetchNetworkPoliciesInNamespace(clusterId, namespaceId) {
         method: 'GET',
         url: `${networkPoliciesBaseUrl}?cluster_id=${clusterId}&namespace=${namespaceId}`,
     };
-    return axios(options).then((response) => ({
-        response: response.data,
-    }));
+    return axios(options).then((response) => response.data.networkPolicies);
```

Comment:
This is only in use in one place, so instead of deep wrapping `{ response: { networkPolicies: NetworkPolicy[] } }` we just return the data back directly as `NetworkPolicy[]`.

Reply:
Yes, please. That was convention of sagas. No more network sagas, no more wrapping.